Source/WebCore/platform/graphics/texmap/TextureMapperLayer.h

```diff
@@ -109,6 +116,11 @@ class WEBCORE_EXPORT TextureMapperLayer : public CanMakeWeakPtr<TextureMapperLay
 
     void addChild(TextureMapperLayer*);
 
+    void acceptDamageVisitor(TextureMapperLayerDamageVisitor *);
```

Comment:
```suggestion
    void acceptDamageVisitor(TextureMapperLayerDamageVisitor*);
```

Reply:
Should we use a reference instead of a pointer?

Reply:
done